frontend/src/components/Course/Grading/NeedHelp.tsx

```diff
@@ -0,0 +1,39 @@
+const NeedHelp = () => {
+    return (
+        <div className="bg-gradient-to-r from-blue-300 to-blue-500 text-white p-8 rounded-lg my-4 w-full text-justify">
+            <h2 className="text-3xl font-bold mb-6">Welcome to the Arbitre Grading System!</h2>
+            <p className="text-lg mb-6">
+                With our incredible application, you can quickly and easily obtain the grade for each session and exercise.
+                Follow these simple steps to get started:
```

Comment:
I don't think we are trying to advertise Arbitre here, in this modal.
Maybe we should stick to simplicity and get straight to the point. 

I think a neutral-toned title could suffice, such as "How to grade courses".

Reply:
Done